docs/pages/blog/mui-x-v8-alpha-zero.md

```diff
@@ -0,0 +1,96 @@
+---
+title: The road to MUI X v8
+description: Explore the planned features and our roadmap to the next major version.
+date: 2024-11-20T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'Product']
+manualCard: true
+---
+
+We're kicking off the development of [MUI X v8](https://github.com/mui/mui-x/releases/tag/v8.0.0-alpha.0).
+Following our yearly release cycle, the target for the first stable release is March 2025.
+This major update includes new versions of the Data Grid, Charts, Tree View, and Date & Time Pickers.
+We’re excited to share our roadmap with you and invite you to join us on this journey!
+
+:::warning
+Only MUI X is getting a new version—Material UI will remain in v6 for now.
+You can rest assured that all MUI X v8 components will be fully compatible with Material UI v5 and v6.
+::::
+
+## What's the plan to get to the next major release?
+
+Before the official release of MUI X v8, we’ll go through a few months of pre-releases.
+During this period, we’ll continue with our usual weekly release process and follow [semantic versioning](https://semver.org/).
+While we’ll still release bug fixes for v7, these will become less frequent as we progress toward a stable v8 release.
+Our primary focus will be on delivering the new features and implementing the necessary breaking changes in the v8 pre-release packages.
+
+In the alpha phase, we’ll introduce all planned breaking changes for this major version. Some APIs may be unstable—not in terms of functionality, but there may be adjustments or renaming of parameters. This phase is expected to last about two to three months.
+
+Following the alpha phase, the beta phase will focus on stabilizing the APIs, fixing bugs, and refining the overall experience. This phase is anticipated to take about one month.
+
+Finally, we'll release the first `v8.0.0` stable by March 2025, and we'll continue improving the components and adding features, but without making any new breaking changes until the next major version.
+
+## What happens to v7?
+
+During v8 pre-releases, v7 will continue to be the official current major, and it will remain supported during this time.
+However, we will only release new v7 versions as needed to patch bugs or add community contributions.
+
+All new features and enhancements will go to v8, and after its first stable release, v7 will officially transition to [long-term support](https://mui.com/x/introduction/support/#long-term-support-lts) status.
+
+## Where's the v8 documentation?
+
+As mentioned above, v7 is still the official version, so by default, the documentation shows v7 features and API.
+The next version's documentation is in the `v8` subdomain.
+
+- [https://v8.mui.com/x/](https://v8.mui.com/x/)
+
+## What's coming next?
+
+The following is a list of enhancements in the pipeline for v8.
+It contains the highlights that will be included in the first versions.
```

Comment:
In case we change the plan? https://youtu.be/Vlph3dn4jnU?si=g5bc8gUV3QB4vvk1&t=4471

```suggestion
It contains the highlights of the improvement that we aim to include in the first versions.
```